tests/e2e/workloads/pgsql/test_pgsql.py

```diff
@@ -38,9 +40,20 @@ def test_sql_workload_simple(self, pgsql):
             replicas=3, clients=3, transactions=600
         )
 
+        # Start measuring time
```

Comment:
You are measuring the time in all the tests.
 can we move this to wait_for_pgbench_status()?